### PR TITLE
bump version 1.5.1 

### LINKED ISSUE #29: [Feat] - Add docker

### PR DESCRIPTION
**Pull request by bogdzn**
- closes #29 

also added fancy little help messages for Makefile: 

20220120_12h09m40s_grim

- bumping to version 1.5.1
- when starting `nuclei`, we now try to look if templates are installed. If they're not, we frist install them using `-update-templates`

### CODE REVIEW

**Comment by jenaye** [comment]
image

We need to go get nuclei

**Comment by jenaye** [comment]
we also need to install google chrome 
image

File: .goreleaser.yml [hunk 1]
```diff
@@ -5,6 +5,13 @@ builds:
 - <<: &build_defaults
     ldflags:
       - -s -w
+  ignore:
+    - goos: freebsd
+      goarch: arm64
+    - goos: windows
+      goarch: arm
+    - goos: windows
+      goarch: arm64
```
Comment (jenaye) [inline .goreleaser.yml:14]: I agree to stop maintaining the ARM windows version

File: .goreleaser.yml [hunk 2]
```diff
@@ -5,6 +5,13 @@ builds:
 - <<: &build_defaults
     ldflags:
       - -s -w
+  ignore:
+    - goos: freebsd
```
Comment (jenaye) [inline .goreleaser.yml:9]: I think we should keep the ARM version for freebsd

File: README.md [hunk 3]
```diff
@@ -24,11 +24,17 @@ Or if you have set your GO path and all the requirements installed :
 go install github.com/CMEPW/Yelaa@latest
 ```
 
-# How to use 
+Using Docker:
+```bash
+make docker # or `docker build . -t yelaa-local`
+docker run -v $PWD:/mnt/ yelaa-local scan -t /mnt/target.txt
```
Comment (jenaye) [inline README.md:30]: we also need to get the screenshots made by the application

File: Dockerfile [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+FROM golang:1.16-alpine as otp-builder
+
+WORKDIR /build
+
+RUN apk update --no-cache && \
+    apk upgrade --no-cache && \
+    apk add --no-cache \
+    make \
+    build-base
+
```
Comment (jenaye) [inline Dockerfile:13]: RUN go install -v github.com/projectdiscovery/nuclei/v2/cmd/nuclei@latest

**Review by jenaye — CHANGES_REQUESTED** [review]

**Review by darkweak — APPROVED** [review]
LGTM

File: Dockerfile (hunk 4 above)
Comment (darkweak) [reply Dockerfile:13]: Not anymore.

**Pull request merged**